### PR TITLE
adding license information to gui client page

### LINKED ISSUE #381: Add licence information in `GUI Clients` page.
`Price: Free` is not adequate enough, we need to add the licence information for each GUI client. 

I am willing to send a PR

### PR DESCRIPTION
**Pull request by alpreu**
Added the license to each one of the GUI Clients under the Download page.
This might close #381

### CODE REVIEW

File: app/views/downloads/guis/index.html.haml [hunk 1]
```diff
@@ -28,109 +28,127 @@
...
+          <strong>Price:</strong> $29/user (Free 15 day trial)</br>
+          <strong>License:</strong> Proprietary
       %li.linux
         =link_to(image_tag('guis/giggle@2x.png', {:width => '294', :height => '166'}), "https://wiki.gnome.org/Apps/giggle/")
         %h4= link_to "giggle", "https://wiki.gnome.org/Apps/giggle/"
         %p.description
           <strong>Platforms:</strong> Linux</br>
-          <strong>Price:</strong> Free
+          <strong>Price:</strong> Free</br>
+          <strong>License:</strong> GNU GPL
```
Comment (pedrorijo91) [inline app/views/downloads/guis/index.html.haml]: two more other GUIs were added in https://github.com/git/git-scm.com/pull/1029 and https://github.com/git/git-scm.com/pull/782, can you rebase to solve the conflicts and add the information for those new GUIs please?

Comment (pedrorijo91) [reply app/views/downloads/guis/index.html.haml]: useful info:

* https://github.com/FredrikNoren/ungit#license-mit
* https://github.com/FabriceSalvaire/CodeReview/blob/master/GPL-V3.0.txt

**Comment by peff** [comment]
I like this conceptually, but I'd want to make sure that the information is accurate, and that the authors of the tools have a chance to respond (e.g., some of the ones marked as proprietary may be in dual license situations, or may want to clarify that they're free for personal use, etc).

I wonder if we could ping the original authors of each block by looking through the output of `git blame`.

**Comment by alpreu** [comment]
@pedrorijo91 I'm not so experienced with git, I tried to rebase, I'm not sure if it worked as you intended. Any help would be appreciated.

@peff Yes, It is hard to have correct information as not every client has clear terms. It is also possible that a license, especially for a commercial product , changes in the future.

**Comment by pedrorijo91** [comment]
The PR is in a weird state @alpreu, it's bringing a lot of changes that have been committed by other contributors

Since your changes are in a single commit (b4efdf1) my recommendation would be to cherry-pick it. No problem if you are not comfortable with git yet :)

Steps)

1. Go to your cloned git-scm repository on your pc
2. Add the original repository (this one) as a remote (a remote is simply a 'source' where you can get or push changes for that repository). You can do this using: `git remote add <someName> git@github.com:git/git-scm.com.git` (we typically call `origin` to our own fork, and `upstream` to the original one). If you are not pushing your git changes through ssh protocol (are using github username/password), type this instead:  `git remote add <someName> https://github.com/git/git-scm.com.git`
3. Now get the recent changes from this repo into your machine by typing: `git fetch <someName>`
4. Now create a new updated branch by typing: `git checkout -b <newBranchName> <someName>/master` (this will say to create a new branch based on the master branch of this repo)
5. finally, get the commit you did into this branch: `git cherry-pick b4efdf1`
6. Now just solve the conflicts, and do a normal commit. You can then create a new (clean) PR and forget about this one

I this this is easier than trying to fix the state of this PR if you are not very comfortable with git yet

**Pull request closed without merging**